src/main/java/lotto/domain/Money.java

```diff
@@ -0,0 +1,61 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class Money {
+    private static final int ZERO = 0;
+    private final int value;
+
+    private Money(int value) {
+        validate(value);
+        this.value = value;
+    }
+
+    private void validate(int value) {
+        if (value < ZERO) {
+            throw new IllegalArgumentException(String.format("돈은 %d보다 작을 수 없습니다.",ZERO));
+        }
+    }
+
+    public static Money from(int money) {
+        return new Money(money);
+    }
+
+    public boolean isMoreThanOrEqual(Money money) {
+        return this.value >= money.value;
+    }
+
+    public Money plus(Money money) {
+        return new Money(this.value + money.value);
+    }
+
+    public Money minus(Money money) {
+        return new Money(this.value - money.value);
+    }
+
+    public Money multiply(int count) {
+        return new Money(this.value * count);
+    }
+
+    public double calculateLottoYield(Money winningReward) {
+        return (double) this.value / winningReward.value;
+    }
```

Comment:
Money 라는 돈에 관한 객체가 로또 수익률을 계산을 하는 것이 조금 어색하다고 생각되는데, 승인님의 생각은 어떠신가요? 
Money 는 돈을 계산하는(나누는) devide 라는 기능을 수행하고, Money.divide() 를 호출하는 측에서 `로또의 수익률을 계산한다` 라는 기능을 가지면 좋을 것 같아요.
이렇게 되면 로또 수익률 뿐만 아니라 다른 계산에서도 Money.devide()를 사용할 수도 있겠네요 😄 
```suggestion
    public double divide(Money winningReward) {
        return (double) this.value / winningReward.value;
    }
```

Reply:
돈 객체는 돈과 관련된 계산만 처리를하고 WinningStatus에서 수익률이나 수익금에 대한 기능을 가지는게 맞는 것 같네요. 감사합니다 :)